x-pack/plugins/alerting/server/plugin.test.ts

```diff
@@ -21,6 +21,13 @@ import { eventLogMock } from '@kbn/event-log-plugin/server/mocks';
 import { actionsMock } from '@kbn/actions-plugin/server/mocks';
 import { dataPluginMock } from '@kbn/data-plugin/server/mocks';
 import { monitoringCollectionMock } from '@kbn/monitoring-collection-plugin/server/mocks';
+import { spacesServiceMock } from '@kbn/spaces-plugin/server/spaces_service/spaces_service.mock';
+
+function createStartMock() {
+  return {
+    spacesService: spacesServiceMock.createStartContract(),
+  };
+}
```

Comment:
**nit:** we should be able to use the mocks exported at the root of the spaces plugin and use `spacesMock.createStart()` in the calls below.

```suggestion
import { spacesMock } from '@kbn/spaces-plugin/server/mocks';
```

Reply:
done :)